babel/src/test/resources/sql/redshift.iq

```diff
@@ -603,11 +603,11 @@ EMPNO, EXPR$1
 select dense_rank() over () from emp where deptno = 30;
 EXPR$0
 6
-6
-6
-6
-6
-6
+1
```

Comment:
What does dense_rank without ORDER BY mean?
I guess it should be an error.

Reply:
It is an error in Calcite (I checked), but not in Redshift.

Reply:
Is Redshift different from PostgreSQL?

```sql
select dense_rank() over() from (select * from generate_series(1,5)) y
```

returns 1,1,1,1,1;

It is really strange to see `1,2,3,4,5` from `dense_rank()`

Reply:
I don't know. Since this is babel, the main point of the test is parsing & validation, not execution. I don't much care if the results are wrong.